scripts/multisrc/lightnovelwp/template.ts

```diff
@@ -0,0 +1,210 @@
+        break;
+      default:
+        novel.status = NovelStatus.Unknown;
+        break;
+    }
+
+    let details = $("div.serl > span")
+    if (!details.length) details = $("div.spe > span");
+    details.each(function () {
+      const detailName = $(this).contents().first().text().replace(":", "").trim().toLowerCase();
+      const detail = $(this).contents().last().text().trim().toLowerCase();
+
+      switch (detailName) {
+        case "الكاتب":
+        case "author":
+        case "auteur":
+          novel.author = detail;
+          break;
+        case "الحالة":
+        case "status":
+          switch (detail) {
+            case "مكتملة":
+            case "completed":
+              novel.status = NovelStatus.Completed;
+              break;
+            case "مستمرة":
+            case "ongoing":
+              novel.status = NovelStatus.Ongoing;
+              break;
+            case "متوقفة":
+            case "hiatus":
+              novel.status = NovelStatus.OnHiatus;
+              break;
+            default:
+              novel.status = NovelStatus.Unknown;
+              break;
+          }
+          break;
+        case "الفنان":
+        case "artist":
+        case "artiste":
+          novel.artist = detail;
+          break;
+      }
+    });
+
+    let genres = $(".sertogenre")
+    if (!genres.length) genres = $(".genxed")
+    novel.genres = genres
+      .children("a")
+      .map((i, el) => $(el).text())
+      .toArray()
+      .join(",");
+
+    let summary = $(".sersys > p").map((i, el) => $(el).text().trim()).toArray();
+    if (!summary.length) summary = $(".entry-content > p").map((i, el) => $(el).text().trim()).toArray();
+    novel.summary = summary.join("\n");
+
+    const chapters: Plugin.ChapterItem[] = [];
+    $(".eplister li").each((i, elem) => {
+      const chapterName = $(elem).find(".epl-num").text() + " " + $(elem).find(".epl-title").text();
+      const chapterUrl = $(elem).find("a").attr("href") || "";
+      const releaseTime = $(elem).find(".epl-date").text().trim();
+      let isFreeChapter: boolean;
+      switch ($(elem).find(".epl-price").text().trim().toLowerCase()) {
+        case "free":
+        case "gratuit":
+        case "مجاني":
+        case "":
+          isFreeChapter = true;
+          break;
+        default:
+          isFreeChapter = false;
+          break;
+      }
+      if (isFreeChapter)
+        chapters.push({
+          name: chapterName,
+          path: chapterUrl.replace(this.site, ""),
+          releaseTime: releaseTime,
+        });
+    });
+
+    if (this.options?.reverseChapters && chapters.length) chapters.reverse();
+
+    novel.chapters = chapters;
+    return novel;
+  }
+
+  async parseChapter(chapterPath: string): Promise<string> {
+    const $ = await this.getCheerio(this.site + chapterPath, false);
+    const chapterText = $(".epcontent p").map((i, el) => $(el)).toArray().join("\n") || "";
+    return chapterText;
```

Comment:
the first part is not useful because i only grab the ".epcontent p"
and the ignore I'm scared to put it to every plugins so i just added an `if (this.id == "kolnovel")`

but thanks I can't read Arabic so i didn't know

Reply:
yes, this was done similarly in main by using  ``if (sourceId === 53)`` so that is good